src/world/World.js

```diff
@@ -514,14 +524,18 @@ World.prototype.step = function(dt, timeSinceLastCalled, maxSubSteps){
     }
 };
 
-/**
- * Step the simulation
- * @method step
- * @param {Number} dt
- */
```

Comment:
I think this is an accidental duplicate of [the existing document block for the step method](https://github.com/brianpeiris/cannon.js/blob/event-docs/src/world/World.js#L453-L468).